packages/unenv-preset/package.json

```diff
@@ -49,7 +49,7 @@
 	},
 	"peerDependencies": {
 		"unenv": "2.0.0-rc.21",
-		"workerd": "^1.20250924.0"
+		"workerd": "^1.20250927.0"
```

Comment:
Everything else in this PR looks like a direct revert of the original PR. But this is moving the workerd version forward, right?